zhaquirks/tuya/builder/__init__.py

```diff
@@ -128,6 +136,20 @@ def tuya_ias(
         self.adds(ias_cfg)
         return self
 
+    def tuya_contact(
+        self,
+        dp_id: int,
+        ias_cfg: TuyaLocalCluster = TuyaIASContact,
+    ) -> QuirkBuilder:
+        """Add a Tuya contact switch configuration."""
+        self.tuya_dp(
+            dp_id,
+            ias_cfg.ep_attribute,
+            IasZone.AttributeDefs.zone_status.name,
```

Comment:
I'll need to check this later, but I think we should forward the correct type here, instead of whatever Tuya sends.
```suggestion
            IasZone.AttributeDefs.zone_status.name,
            converter=lambda x: IasZone.ZoneStatus.Alarm_1 if x != 0 else 0,
```

Reply:
ok, will try that